builder/builder.py

```diff
@@ -233,14 +233,18 @@ def __generate_archive(self, build_id: str) -> None:
             build_info.board,
             "bin"
         )
-        bin_list = os.listdir(bin_path)
-        self.logger.debug(f"bin_path: {bin_path}")
-        self.logger.debug(f"bin_list: {bin_list}")
-        for file in bin_list:
-            file_path_abs = os.path.abspath(
-                os.path.join(bin_path, file)
-            )
-            files_to_include.append(file_path_abs)
+
+        if os.path.isdir(bin_path):
```

Comment:
Any reason to no mkpath instead?

Reply:
Oh yes, that's for suggesting. I have pushed a change. Is this what you were looking for?